src/com/palmergames/bukkit/towny/listeners/TownyWorldListener.java

```diff
@@ -205,4 +207,18 @@ public void onPortalCreate(PortalCreateEvent event) {
 		}
 	}
 
+	@EventHandler(priority = EventPriority.NORMAL)

```

Comment:
I haven't yet seen if this PR is necessary after the api/damage_events branch is merged but here's a replacement block for the code below, which will cover both the wilderness and non-wilderness, as well as handle worlds where Towny isn't active or when Towny doesn't load properly:

```java
	@EventHandler(priority = EventPriority.NORMAL)
	public void onLightningStrike(LightningStrikeEvent event) {
		
		if (plugin.isError()) {
			event.setCancelled(true);
			return;
		}

		if (!TownyAPI.getInstance().isTownyWorld(event.getWorld()))
			return;
		
		TownyWorld townyWorld = null;
		try {
			townyWorld = TownyUniverse.getInstance().getDataSource().getWorld(event.getLightning().getWorld().getName());
		} catch (NotRegisteredException ignored) {}
		
		if (townyWorld != null && event.getCause().equals(LightningStrikeEvent.Cause.TRIDENT)) {
			event.setCancelled(CombatUtil.preventPvP(townyWorld, TownyAPI.getInstance().getTownBlock(event.getLightning().getLocation())));
		}
	}
```